airflow/providers/google/CHANGELOG.rst

```diff
@@ -23,6 +23,16 @@
 Changelog
 ---------
 
+9.1.0
```

Comment:
```suggestion
10.0.0
```
breaking changes in provider requires major release

Reply:
I was not making any changes in the logic, just changed the default version, so picked next minor version from the list